src/main/scala/org/scalacheck/ScalaCheckFramework.scala

```diff
@@ -178,20 +182,20 @@ final class ScalaCheckFramework extends Framework {
 
   val name = "ScalaCheck"
 
-  def fingerprints: Array[Fingerprint] = Array(
+  def fingerprints: Array[Fingerprint | Null] = Array(
     mkFP(false, "org.scalacheck.Properties"),
     mkFP(false, "org.scalacheck.Prop"),
     mkFP(true, "org.scalacheck.Properties"),
     mkFP(true, "org.scalacheck.Prop")
   )
 
-  def runner(_args: Array[String], _remoteArgs: Array[String],
-    _loader: ClassLoader
+  def runner(_args: Array[String | Null] | Null, _remoteArgs: Array[String | Null] | Null,
+    _loader: ClassLoader | Null
   ): Runner = new ScalaCheckRunner {
 
-    val args = _args
-    val remoteArgs = _remoteArgs
-    val loader = _loader
+    val args = _args.nn.map(._nn)
```

Comment:
Argument should never be null, immediately remove nullness of parameter.
LOC changed: 3